### PR TITLE
feat: stargate: Enable integration tests with panacea-core docker

### PR DESCRIPTION
**Pull request by youngjoon-lee**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,20 +10,43 @@ on:
...
 
 jobs:
-  build:
+  build-test:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
```
Comment (youngjoon-lee) [inline .github/workflows/ci.yml:19]: Let's run tests only with Node.js 14.x (for simplifying the docker orchestration)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -10,20 +10,43 @@ on:
...
+    env:
+      CHAIN_ID: testing
+      MNEMONIC: ${{ secrets.TEST_MNEMONIC }}
```
Comment (youngjoon-lee) [inline .github/workflows/ci.yml:19]: This loads a secret from the Github repo: https://docs.github.com/en/actions/reference/encrypted-secrets

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -10,20 +10,43 @@ on:
...
+    - name: Run a panacea-core docker container
+      run: |
+        docker run --rm -d \
+          -e CHAIN_ID="${CHAIN_ID}" \
+          -e MNEMONIC="${MNEMONIC}" \
+          -p 26657:26657 \
+          -v $(pwd)/scripts:/root/scripts \
+          --name core \
+          ghcr.io/medibloc/panacea-core:master \
+          bash /root/scripts/panacea-core/init.sh
```
Comment (gyuguen) [inline .github/workflows/ci.yml:44]: I think it's a good way.
I'll have to modify the `panacea-java` similarly later.
Thank you!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (gyuguen) [reply .github/workflows/ci.yml:19]: Oh, Can not looking `TEST_MNEMONIC`.
It's looks good to put confidential data in.

File: scripts/panacea-core/init.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# Initialize a chain
+panacead init node1 --chain-id=${CHAIN_ID}
+echo -e "${MNEMONIC}\n\n" | panacead keys add validator -i
+panacead add-genesis-account $(panacead keys show validator -a) 100000000000000umed
+panacead gentx validator 1000000000000umed --commission-rate 0.1 --commission-max-rate 0.2 --commission-max-change-rate 0.01  --min-self-delegation 1000000 --chain-id ${CHAIN_ID}
+panacead collect-gentxs
+
+# Open a 26657 RPC port publicly
+sed -i 's|^laddr = "tcp://127.0.0.1:26657"$|laddr = "tcp://0.0.0.0:26657"|g' ~/.panacea/config/config.toml
+sed -i 's|^cors_allowed_origins = .*|cors_allowed_origins = \["*"\]|g' ~/.panacea/config/config.toml
```
Comment (gyuguen) [inline scripts/panacea-core/init.sh:16]: This part of the problem that hang on earlier.

**Review by gyuguen — APPROVED** [review]
LGTM. 👍

**Pull request merged**